modules/ROOT/pages/deploy-maven-3.x.adoc

```diff
@@ -81,28 +88,29 @@ This example uses placeholder values. Configure each value with values specific
 
 == Redeploy to Runtime Fabric
 
-To redeploy the application, run the same command as you did to deploy. Runtime Fabric rewrites the application you had deployed.
+To redeploy the application, run the same command as you did to deploy. Runtime Fabric rewrites the 
```

Comment:
"as you did to deploy" -- is there better wording we could use there? 

"as you ran to deploy" is better, IMO.

Reply:
the word "rewrites" is interesting.. all it's doing is re-deploying with the new artifact (the application).

Reply:
Reworded this.